.github/workflows/ci.yml

```diff
@@ -221,16 +213,28 @@ jobs:
       - name: Run tests
         id: run-tests
         run: |
-          docker exec -w /workspace/checkout/${{ matrix.app }} unreal ue4 build
-          docker exec -w /workspace/checkout/${{ matrix.app }} unreal ue4 editor \
+          docker exec -w /workspace/checkout/${{ matrix.app }} unreal /home/ue4/UnrealEngine/Engine/Build/BatchFiles/RunUAT.sh BuildCookRun \
+            -project=/workspace/checkout/${{ matrix.app }}/SentryPlayground.uproject \
+            -archivedirectory=/workspace/checkout/${{ matrix.app }}/Builds \
+            -platform=Linux \
+            -nop4 \
+            -cook \
+            -build \
+            -stage \
+            -prereqss \
+            -package \
+            -archive
+          docker exec -w /workspace/checkout/${{ matrix.app }} unreal bash -c "
+            cp -r '/home/ue4/Library/Logs/Unreal Engine/LocalBuildLogs' Saved/Logs  "
+          docker exec -w /workspace/checkout/${{ matrix.app }} unreal /home/ue4/UnrealEngine/Engine/Binaries/Linux/UnrealEditor \
             /workspace/checkout/${{ matrix.app }}/SentryPlayground.uproject \
             -ReportExportPath=/workspace/checkout/${{ matrix.app }}/Saved/Automation \
             -ExecCmds="Automation RunTests Sentry;quit" \
             -TestExit="Automation Test Queue Empty" \
             -Unattended \
             -NoPause \
             -NoSplash \
-            -NullRHI
+            -NullRHI          
```

Comment:
Could you explain what's going on here? If I read that right:
1. We're first building the project via the `RunUAT.sh` but with **all** the flags.
2. We're copying some logs to somewhere else
3. We're running some tests but from within the Editor
Do I get that right?